### ISSUE TITLE
How to Lossless Reduce the Quantity of VMStorage?

### ISSUE THREAD

**Issue by thislyc** [VictoriaMetrics/VictoriaMetrics#6225]
### Is your question request related to a specific component?

vmstorage

### Describe the question in detail

In the VictoriaMetrics cluster version, I can easily expand the number of vmstorage, but when my resources are low, how can I reduce the number of vmstorage without any anomalies in the original data? I hope there is a tool that can help me merge data from old VMstorage, so that I can easily reduce VMstorage nodes while the cluster is working properly

### Troubleshooting docs

- [X] General - https://docs.victoriametrics.com/troubleshooting/
- [X] vmagent - https://docs.victoriametrics.com/vmagent/#troubleshooting
- [X] vmalert - https://docs.victoriametrics.com/vmalert/#troubleshooting

**Comment by jiekun** [comment]
I remember that I had some discussion in Slack about downscaling: https://victoriametrics.slack.com/archives/CTC488T1P/p1713934149505319

TL;DR (and cmiiw): Not possible (in a non-manual way) yet. 

To maintainers: Should we consider clarifying it in the doc, since similar questions have appeared a couple of times?

**Comment by jiekun** [comment]
I'm quoting my previous answer in Slack:
> (just FYI and it's not the official answer)
And one possible way (manually) to scale down while providing availability + without data migration might be:
> 
> 1. update `vminsert` config, write data to 1st and 2nd node, remove the 3rd node;
> 2. keep `vmselect` config as is, and wait until all the data in 3rd `vmstorage` are expired.
> 3. now it should be safe to adjust `replicaCount` to 2.

So basically you will need to wait for a long time.
